ClientApp/src/components/CardPage.jsx

```diff
@@ -107,6 +152,74 @@ export class CardPage extends Component {
     }
 
     /* Call your backend here */
+    async maledictionHandler() {
+        let targetIndex = this.state.formControls.mTarget.value;
+        let target = this.state.users[targetIndex];
+
+        await fetch('api/card/malediction/' + target.id + '/' + getWeekSince(),
+            {
+                method: 'GET',
+                headers: {
+                    ...{
+                        'Accept': 'application/json',
+                        'Content-Type': 'application/json'
+                    },
+                    ...authHeader()
+                }
+            }).then(
+            window.location.reload()
+        );
+    }
+
+    async doubleSendHandler() {
```

Comment:
For this and revive, it might be  better to add a submit button or a dialog box when the user clicks the card ( what if he clicks the card expecting more info like the other cards and does not want to use it immediately?). It's better for consistency reasons and to give the user a chance to think before using it.